doc/source/api.rst

```diff
@@ -0,0 +1,40 @@
+API
+---
+
+Particle data
+=============
+
```

Comment:
Please add short summary paragraphs for the sections of the API. I did this for the topology section.